### ISSUE TITLE
Does not know how to "rake curation_concerns:jetty:config" as stated in README.md

### ISSUE THREAD

**Issue by ravn** [pulibrary/plum#421]
Trying to get plum up and running as of today on Ubuntu 15.10.

```
tra@PC599:~/git/plum$ rake curation_concerns:jetty:config
rake aborted!
Don't know how to build task 'curation_concerns:jetty:config' (see --tasks)

(See full trace by running task with --trace)
```

It appears that the newly released 0.8.0 is used of curation concerns, and this task is not listed with --tasks.

Please advise.

**Comment by escowles** [comment]
@ravn, it looks like the migration to Solr 5 has broken the configuration somewhere.  I'll work on updating the docs so they work with the new Solr setup.

**Comment by ravn** [comment]
Sounds great - is there a simple workaround I can use for now?

**Comment by escowles** [comment]
@ravn, I've just pushed a branch named `solr5`, with new Rake tasks for running the Fedora/Solr servers for development and test modes.  Can you test the updated instructions in the README and make sure they work for you?

**Comment by ravn** [comment]
Thank you.

I have now followed the instructions on a Ubuntu 14.04 LTS box and gotten the system up and running (but then stuck on the CAS login, but that is another matter).  

Is the Tesseract 3.04 requirement very hard or would the 3.03 which is available in the Ubuntu repositories for 14.04 be ok?

Thank you again.

**Comment by escowles** [comment]
You should be able to use Tesseract 3.03.  We started out using a gem with native bindings that tied us tightly to the particular version, but we switching to just shelling out, so that will be much more accepting of different versions.

Being more flexible for the auth is on the todo-list: #263.  Hopefully it won't be too hard to swap in something else, e.g., Omniauth's developer module for the development environment.
